test/parallel/test-http-allow-req-after-204-res.js

```diff
@@ -23,12 +23,14 @@
 const common = require('../common');
 const http = require('http');
 const assert = require('assert');
+const Countdown = require('../common/countdown');
 
 // first 204 or 304 works, subsequent anything fails
 const codes = [204, 200];
 
 // Methods don't really matter, but we put in something realistic.
 const methods = ['DELETE', 'DELETE'];
+const countdown = new Countdown(methods.length, () => server.close());
```

Comment:
Can you please wrap the callback into `common.mustCall()`?

Something like this:

```js
const countdown = new Countdown(methods.length, common.mustCall(() => {
  server.close();
}));